Refresh.GameServer/Endpoints/ApiV3/DataTypes/Request/ApiUpdateUserRequest.cs

```diff
@@ -14,4 +16,7 @@ public class ApiUpdateUserRequest
     public bool? UnescapeXmlSequences { get; set; }
     
     public string? EmailAddress { get; set; }
+    
+    public Visibility? LevelVisibility { get; set; }
+    public Visibility? ProfileVisibility { get; set; }
```

Comment:
New properties in this class should be handled in the corresponding function in `GameDatabaseContext`. The function in question is the `ApiUpdateUserRequest` overload of `UpdateUserData`.

This makes it so these properties can actually be updated from the API - we removed the reflection-based updating a while ago.

Reply:
i did this [here](https://github.com/LittleBigRefresh/Refresh/pull/520/files#diff-82503085bd955b6efc6eb6353f5f08e8cb21ac466bb03eaad9d5b0e7f3a0eaf8R181)